### PR TITLE
Update PROJECT_NAME_incubation_stage.md

### PR DESCRIPTION
**Pull request by SecurityCRob**
proposed changes to incubating project template

### CODE REVIEW

File: process/templates/PROJECT_NAME_incubation_stage.md [hunk 1]
```diff
@@ -14,7 +17,19 @@ The project should be able to show adoption by multiple parties and the adoption
 
 ### Governance
 Projects must have documented, initial project governance
-  * "link to governance documents"
+  * "link to governance documents/Charter"
+Projects must have met publicly at least 5 times
+  * "link to meeting agenda"
+Project must have a TAC sponsor
```
Comment (marcelamelara) [inline process/templates/PROJECT_NAME_incubation_stage.md:23]: I'm a bit confused. Per #262, I thought we had relaxed the TAC sponsor requirement for Projects/SIGs, while making it a requirement for WGs at the sandbox stage: https://github.com/ossf/tac/blob/main/process/TI-Gives%2BGets.md#givesrequirements

**Review by marcelamelara — COMMENTED** [review]
Just one point of clarification.

**Review by steiza — APPROVED** [review]
What a process it is rationalizing our lifecycle docs, TI gives and gets, and these templates; thanks for putting this together!

**Review by camaleon2016 — APPROVED** [review]

**Review by steiza — APPROVED** [review]
LGTM!

**Review by bobcallaway — APPROVED** [review]

File: process/templates/PROJECT_NAME_incubation_stage.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 ## Project incubation application
 
+### Project has met all Sandbox requirement
+  * "link to sandbox PR"
```
Comment (lehors) [inline process/templates/PROJECT_NAME_incubation_stage.md:4]: Same comment as before: I think there is no need to ask people to dig this up.

**Review by lehors — CHANGES_REQUESTED** [review]

File: process/templates/PROJECT_NAME_incubation_stage.md (hunk 2 above)
Comment (SecurityCRob) [reply process/templates/PROJECT_NAME_incubation_stage.md:4]: Is it acceptable to leave in the form, but grandfather in existing projects to NOT have to do this, but all NEW work would need to provide the link as they progress?

File: process/templates/PROJECT_NAME_incubation_stage.md [hunk 3]
```diff
@@ -14,7 +17,13 @@ The project should be able to show adoption by multiple parties and the adoption
 
 ### Governance
 Projects must have documented, initial project governance
```
Comment (marcelamelara) [inline process/templates/PROJECT_NAME_incubation_stage.md:19]: Adding this per the TI G&Gs [incubating requirements](https://github.com/ossf/tac/blob/main/process/TI-Gives%2BGets.md#givesrequirements-1).
```suggestion
Project must have met publicly at least 5 times
   * Link to public meeting notes (or ideally recordings)
Projects must have documented, initial project governance
```

**Review by sevansdell — APPROVED** [review]

File: process/templates/PROJECT_NAME_incubation_stage.md (hunk 2 above)
Comment (SecurityCRob) [reply process/templates/PROJECT_NAME_incubation_stage.md]: added in

**Comment by SecurityCRob** [comment]
6 of 9 TAC members approve, merging

**Pull request merged**